### PR TITLE
Clipping Primitives PR

### PR DESCRIPTION
**Pull request by palmerdesigns**
## What's Added?
Scene elements composed of Clipping examples:
- Dinosaur - Clipping Box
- Lava Lamp - Clipping Sphere
- Nested Dolls - Clipping Plane

### CODE REVIEW

**Comment by Cameron-Micka** [comment]
Can you add the Clipping Primitive sample to the list if samples in the package.json? You can see how that is done here: https://github.com/microsoft/MixedReality-GraphicsTools-Unity/pull/87/files#diff-a139dcebf53553d0bffae890c2f82323aaa9a3d1aa500232df9613f0efe01b30 

More details here too: https://docs.unity3d.com/Manual/cus-samples.html

File: com.microsoft.mrtk.graphicstools.unity/Samples~/Clipping Primitives/Scripts/Rotate.cs [hunk 1]
```diff
@@ -0,0 +1,14 @@
+using System.Collections;
```
Comment (Cameron-Micka) [inline com.microsoft.mrtk.graphicstools.unity/Samples~/Clipping Primitives/Scripts/Rotate.cs:1]:
```suggestion
// Copyright (c) Microsoft Corporation.
// Licensed under the MIT License.

using System.Collections;
```

File: com.microsoft.mrtk.graphicstools.unity/Samples~/Clipping Primitives/Scripts/Rotate.cs [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Rotate : MonoBehaviour
```
Comment (Cameron-Micka) [inline com.microsoft.mrtk.graphicstools.unity/Samples~/Clipping Primitives/Scripts/Rotate.cs:5]: Could you put this class in the `Microsoft.MixedReality.GraphicsTools.Samples.ClippingPrimitives` namespace?

File: com.microsoft.mrtk.graphicstools.unity/Samples~/Clipping Primitives/LightMaps/LightingData.asset.meta [hunk 3]
```diff
@@ -0,0 +1,8 @@
+fileFormatVersion: 2
```
Comment (Cameron-Micka) [inline com.microsoft.mrtk.graphicstools.unity/Samples~/Clipping Primitives/LightMaps/LightingData.asset.meta:1]: Would you mind deleting all the lightmap data? If you don't need it we normally don't check it in to reduce space.

**Review by Cameron-Micka — CHANGES_REQUESTED** [review]
Looking really good! Added a few comments below. I'll also include some comments below for things I can't directly reference.

1. Can you replace the camera in the scene with the fly camera? 
image
2. Some of the textures are pretty high res (2k in some cases) can you reduce them? Red pixels in this image mean the texture is higher res than it needs to be: 
image
3. Could you add a description panel simiilar to the other sample scenes? 
image

**Review by Cameron-Micka — CHANGES_REQUESTED** [review]
Looks awesome! The last thing to update before this can be merged is the prefab reference. Each sample needs its own prefab since samples can exist on their own: 
image

**Comment by palmerdesigns** [comment]
Thanks, I'm on it!

File: com.microsoft.mrtk.graphicstools.unity/Samples~/ClippingPrimitives/Prefabs/DescriptionPanel.prefab [hunk 4]
```diff
@@ -0,0 +1,469 @@
+%YAML 1.1
```
Comment (Cameron-Micka) [inline com.microsoft.mrtk.graphicstools.unity/Samples~/ClippingPrimitives/Prefabs/DescriptionPanel.prefab:1]: Looks like the material for the backplate is missing: 
image

File: com.microsoft.mrtk.graphicstools.unity/Samples~/MeshOutlines/Prefabs/DescriptionPanel.prefab [hunk 5]
```diff
@@ -221,7 +221,7 @@ MonoBehaviour:
   m_FallbackScreenDPI: 96
   m_DefaultSpriteDPI: 96
   m_DynamicPixelsPerUnit: 1
-  m_PresetInfoIsWorld: 0
+  m_PresetInfoIsWorld: 1
```
Comment (Cameron-Micka) [inline com.microsoft.mrtk.graphicstools.unity/Samples~/MeshOutlines/Prefabs/DescriptionPanel.prefab:224]: nit pick: could you revert this change?

**Review by Cameron-Micka — CHANGES_REQUESTED** [review]
Added comments for two minor things and then this is ready to merge! 🚀

**Review by Cameron-Micka — APPROVED** [review]
Perfect!
